site/homepage/content/sandbox.md

```diff
@@ -37,7 +37,7 @@ All services are exposed via the same ports as used in the guide.
   We therefore **strongly suggest** that you use the secure ports only, if possible!
   When using the [Hono client](https://www.eclipse.org/hono/docs/latest/admin-guide/hono-client-configuration/) to access the sandbox'
   Telemetry and/or Event APIs, make sure to not set a trust store explicitly but instead set the *tlsEnabled* property to `true`.
-  The command line client binary is available from the [downloads page](https://www.eclipse.org/hono/download/#binaries) and
+  The command line client binary is available from the [downloads page](https://www.eclipse.org/hono/downloads/#binaries) and
```

Comment:
this should better be a relref, I guess ...

Reply:
the link to the documentation two sentences up should also be changed to a %doclink% ...

Reply:
> the link to the documentation two sentences up should also be changed to a %doclink% ...

I would prefer to do this in a separate PR, since I get 64 hits for "eclipse.org/hono/docs" in the `site/homepage/content` folder.